docs/code/MlNetCookBook.md

```diff
@@ -578,6 +578,48 @@ var biases = modelParameters.GetBiases();
 
 ```
 
+## How do I look at the global feature importance?
+The below snippet shows how to get a glimpse of the the feature importance, or how much each column of data impacts the performance of the model.
+
+```csharp
+var transformedData = model.Transform(data);
+
+var featureImportance = context.Regression.PermutationFeatureImportance(model.LastTransformer, transformedData);
+
+foreach (var metricsStatistics in featureImportance)
+{
+    Console.WriteLine($"Root Mean Squared - {metricsStatistics.Rms.Mean}");
+}
+```
+
+## How do I get a model's weights to look at the global feature importance?
+The below snippet shows how to get a model's weights to help determine the feature importance of the model.
+
+```csharp
+var linearModel = model.LastTransformer.Model;
+
+var weights = new VBuffer<float>();
+linearModel.GetFeatureWeights(ref weights);
+```
+
+## How do I look at the feature importance per row?
```

Comment:
"Local feature importance"

"row" => "example" (the language we've shifted to using)